cmd/server/http/describe.go

```diff
@@ -180,7 +180,8 @@ func describeLatestArtifacts(ctx context.Context, payload *payload, flowSvc *flo
 		values := r.URL.Query()
 		branch := values.Get("branch")
 		if emptyString(branch) {
-			branch = "master"
+			httpinternal.Error(w, "missing required branch parameter", http.StatusBadRequest)
```

Comment:
This could be `requiredFieldError` instead?

```suggestion
			requiredFieldError(w, "branch")
```

Reply:
Yes! Was looking for that 🙄 🤔 💪